web/templates/layout/_segment.html.eex

```diff
@@ -0,0 +1,91 @@
+<script type="text/javascript">
```

Comment:
Is there a reason for this to be here specifically and not in `web/static/js/` as a .js file?